Resources/Prototypes/_LostParadise/Entities/Structures/Machines/vending_machines.yml

```
@@ -290,3 +290,36 @@
     energy: 1.6
     color: "#afeeee"
 
+- type: entity
+  parent: VendingMachine
+  id: LPPTeaVend
+  name: TeaVend
+  description: Perfect for those who want to cool down a bit.
+  components:
+  - type: VendingMachine
+    pack: TeaVendingMachineServitorInventory
+    offState: off
```

yamllint

[warning] truthy value should be one of [false, true]

(truthy)